common/src/main/kotlin/flank/common/Files.kt

```diff
@@ -14,6 +15,24 @@ val userHome: String by lazy {
 
 fun String.deleteFile() = Paths.get(this).delete()
 
+fun linkFiles(
+    link: String,
+    target: String
+) = if (isWindows) copyDirectory(target, link) else createSymbolicLink(link, target)
+
+fun copyDirectory(sourceDirectoryLocation: String, destinationDirectoryLocation: String) {
+    Files.walk(Paths.get(sourceDirectoryLocation))
+        .forEach { source: Path ->
+            val destination =
+                Paths.get(destinationDirectoryLocation, source.toString().substring(sourceDirectoryLocation.length))
+            try {
+                Files.copy(source, destination)
+            } catch (e: IOException) {
+                logLn(e.localizedMessage)
+            }
+        }
+}
```

Comment:
Could we add some os based tests to it to just make sure that it creates symbolic link?

Reply:
@piotradamczyk5 there is already test for symbolic linking done previously.

Reply:
its also fixed for windows
i will add tests for new functions added

Reply:
@piotradamczyk5 added